tests/unit/utils/test_modules.py

```diff
@@ -34,3 +36,22 @@ def test_ResidualBlock_shortcut(double_module):
         double_module, shortcut=utils.ResidualBlock(double_module)
     )
     assert model(input) == ptu.approx(5.0)
+
+
+def test_SequentialWithOutChannels(subtests):
+    sequentialpaper = (nn.Conv2d(3, 3, 1), nn.Conv2d(3, 5, 1))
```

Comment:
Could you rename this to sequential_modules? This was accidentally introduced in b02df27b while refactoring.